TestBenches/InputRouter_test.cpp

```diff
@@ -429,19 +472,21 @@ int main(int argc, char * argv[])
         cNonant  = std::stol( cArg.substr( cArg.find(",")+1 , cArg.length()-1) , nullptr, 10 );
         std::cout << "Tk nonant passed from cmd line is " << cDTCsplit << "\n";
       }
+        if( cArg.find("lastBx") != std::string::npos ) 
+      {
+        cLastBx  = std::stol( cArg.substr( cArg.find(",")+1 , cArg.length()-1) , nullptr, 10 );
+        std::cout << "Last Bx to process assed from cmd line is " << cLastBx << "\n";
```

Comment:
@sseifeln I'm guessing "assed" is a typo. Can you fix it before we merge :)

Reply:
🤦 yes.. typo! Removed the option of modifying this from the cmd line (was using it during testing the logic but it shouldn't be needed now)